crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -372,7 +372,7 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
     /// # my_system.system();
     /// ```
     #[inline]
-    pub fn id(&self) -> Entity {
+    pub const fn id(&self) -> Entity {
```

Comment:
`EntityCommands` get created in the `entity()` call of `Commands` which is not const.